### DISCUSSION TITLE
Manifest version

### DISCUSSION THREAD

**Discussion by icweaver** [JuliaAstro/Transits.jl#3] [Q&A] [answered]
Hi all,

Very excited to begin working on this project! I was wondering if anyone was getting this error when attempting to instantiate or test on 1.5.3:

```julia
ERROR: AssertionError: sourcepath !== nothing
```

The problem seems to go away when trying on 1.6.0-rc1, so could it possibly be related to this [issue](https://github.com/julia-actions/julia-runtest/issues/18)?

**Comment by giordano** [comment, marked as answer]
You're instantiating a manifest generated with a different version of Julia

**Reply by icweaver** [reply]
Ugh, I guess that's what I get for flipping back and forth between versions without checking the manifest, thanks Mosè!

Also, I think it is really neat that in 1.6 things are automatically re-resolved if necessary when testing!

**Reply by giordano** [reply]
It's unfortunate, but manifests are julia-version-specific, especially if there are changes in standard libraries (and many new were introduced in v1.6)
